docs/proposals/compute-resource-metrics-api.md

```diff
@@ -115,13 +116,11 @@ only be available through the kubelet. The types of metrics are detailed
 Additionally, all endpoints (except root discovery endpoint) support the
 following optional query parameters:
 
-- `start` - start time to return metrics from; type json encoded
-  `time.Time`; since samples are retrieved at discrete intervals, the first
-  sample after the start time is the actual beginning.
-- `end` - end time to return metrics to; type json encoded `time.Time`
-- `step` - the time step between each stats sample; type int (seconds), default
```

Comment:
Without step or even samples spreading maxSamples > 1 is not particularly useful. Imagine 1 min window and maxSamples set to 3. Having samples at 20s, 40s and 60s (step=20) is much more useful than at 58, 58, 60s. So either drop it entirely (Heapster will most likely set it to 1 anyway) or add step.